name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/messages_de.properties

```diff
@@ -387,9 +387,9 @@ ColumnDividendTotalRateOfReturn = Div%
 
 ColumnDividendTotalRateOfReturn_Description = Dividendenrendite = Summe der Dividendenzahlungen / Einstand nach FIFO\n\nAchtung: wenn Verk\u00E4ufe nach einer Dividendenzahlung vorliegen, werden die momentan nicht abgezogen. D.h. die Rendite ist eventuell zu hoch.
 
-ColumnDividendYieldOnCost = Div%YoC
+ColumnDividendYieldOnCost = Div%YoC 1J
 
-ColumnDividendYieldOnCost_Description = Dividendenrendite nach Einstandskosten (YoC)
+ColumnDividendYieldOnCost_Description = Dividendenrendite nach Einstandskosten (YoC) f�r die letzten 12 Monate.
```

Comment:
Bitte benutze den [ResourceBundle Editor](https://github.com/buchen/portfolio/blob/master/CONTRIBUTING.md#install-eclipse-plugins) in Eclipse für die Erstellung oder Anpassung von *.properties
--> UTF-8 Konvertierung